recipes/transx2gtfs/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "0.4.1" %}
+{% set name = "transx2gtfs" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 23eb656f0c56d550de6a35337a3994aac2d6effd5c9b40351a3fdc46fbe721c8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas
+    - pyproj
+    - untangle
+
+test:
+  imports:
+    - transx2gtfs
```

Can we try to add ``pip check`` for this recipe please?
```suggestion
    - transx2gtfs
  requires:
    - pip
  commands:
    - pip check
```

---

Sure. 👍